tests/integration/security/testdata/authz/v1beta1-tcp.yaml.tmpl

```diff
@@ -8,19 +8,19 @@ metadata:
 spec:
   selector:
     matchLabels:
-      "app": "b"
+      "app": "{{ .b }}"
```

Comment:
can you also include `{{ .b }}` in the name of the policy? Same here and below for c, d, e.

Reply:
Updated.